rust/src/wrapper/hci.rs

```diff
@@ -134,12 +112,45 @@ impl Address {
     }
 }
 
-/// BT address types
-#[allow(missing_docs)]
-#[derive(PartialEq, Eq, Debug)]
-pub enum AddressType {
-    PublicDevice,
-    RandomDevice,
-    PublicIdentity,
-    RandomIdentity,
+impl ToPyObject for Address {
+    fn to_object(&self, _py: Python<'_>) -> PyObject {
+        self.0.clone()
+    }
+}
+
+/// Implements minimum necessary interface to be treated as bumble's [HCI_Command].
+/// While pyo3's macros do not support generics, this could probably be refactored to allow multiple
+/// implementations of the HCI_Command methods in the future, if needed.
+#[pyclass]
+pub(crate) struct HciCommandWrapper(pub(crate) Command);
+
+#[pymethods]
+impl HciCommandWrapper {
+    fn __bytes__(&self, py: Python) -> PyResult<PyObject> {
```

Comment:
whats the meaning of the double underscores ?

Reply:
This is the Python function that gets called internally when a caller (in Python) invokes `bytes(some_obj)` (https://docs.python.org/3/reference/datamodel.html?highlight=__bytes__#object.__bytes__)

As to why Python uses double underscore before and after such methods... Probably to discourage calling them directly? They're typically magic methods that are special to Python.